app/views/api/_location.json.builder

```diff
@@ -1,15 +1,18 @@
 json.id               object.id
 json.type             "location"
-json.name             object.text_name
-json.latitude_north   object.north
-json.latitude_south   object.south
-json.longitude_east   object.east
-json.longitude_west   object.west
-json.altitude_maximum object.high
-json.altitude_minimum object.low
-json.notes            object.notes.to_s.tpl_nodiv
+json.name             object.text_name.to_s
+json.latitude_north   object.north if object.north.present?
```

Comment:
Would this work & be more efficient? (No reply necessary)
```suggestion
json.latitude_north   object.try(:north)
```

Reply:
Probably more efficient, but specifically my point in adding all of these "if object.xxx.present?" constructs was to reduce the length of the output.  There is no reason to include nulls or empty strings in the JSON or XML output.  As the vast majority of observations do not include all fields (e.g., obscure things like external_links, or even sequences, herbarium records, etc. etc.) this reduces the overall length of the response our server needs to send to the client radically.  When you're dealing with data on 1000 observations, that adds up significantly.